melodic/distribution.yaml

```diff
@@ -11680,6 +11680,12 @@ repositories:
       url: https://github.com/ros-visualization/rqt_nav_view.git
       version: master
     status: maintained
+  rqt_paramedit:
+    doc:
```

Comment:
Could you add also a `source` entry?
Having only a docs entry is fine though, but it's more common to have both.

Reply:
I just added a source tag.